doc/developer/debugging.rst

```diff
@@ -166,39 +196,26 @@ Add a ``docker-compose.override.yml`` file with a ``geoportal`` service containi
          - <c2cgeoportal_git_root>/geoportal/c2cgeoportal_geoportal:/opt/c2cgeoportal_geoportal/c2cgeoportal_geoportal
          - <c2cgeoportal_git_root>/admin/c2cgeoportal_admin:/opt/c2cgeoportal_admin/c2cgeoportal_admin
 
-Expose a service
-................
+Access to a hidden service
+--------------------------
+
+Within the Docker composition you can access a port of a container, you can achieve this via curl, e.-g.:
+
+.. prompt: bash
 
-To expose a service out of the Docker composition you can add a port in your ``docker-compose.yaml``, e.g.:
+   curl "http://mapserver:8080?SERVICE=WMS&VERSION=1.1.1&REQUEST=GetCapabilities"
+
+You can also expose a service out of the Docker composition you can add a port in your
```

Comment:
You can also expose a service out of the Docker composition. For that, add a port in your